src/scenic/simulators/newtonian/simulator.py

```diff
@@ -149,9 +169,14 @@ def addRegion(region, color, width=1):
 
     def scenicToScreenVal(self, pos):
         x, y = pos[:2]
-        x_prop = (x - self.min_x) / self.size_x
-        y_prop = (y - self.min_y) / self.size_y
-        return int(x_prop * WIDTH), HEIGHT - 1 - int(y_prop * HEIGHT)
+
+        screen_x = int((x - self.min_x) * self.screenScaling)
+        screen_y = HEIGHT - 1 - int((y - self.min_y) * self.screenScaling)
+
+        screen_x = screen_x + self.screenTranslation[0]
+        screen_y = screen_y + self.screenTranslation[1]
```

Comment:
These will be floats in general, not ints, but I guess that doesn't break pygame?

Reply:
Ah good catch, I'll cast them.